File: CPP/Day9.cpp
/*

Day9: Program to find factorial of given number

*/


//Program
#include <iostream>
using namespace std;

int main()
{
    int n, fact=1;
    cout << "Enter a number: ";
    cin >> n;
    for(int i=1; i<=n; i++)
    {
        fact = fact * i;
    }
    cout<< "Factorial of "<< n << " is "<< fact << endl;

    return 0;
}


/*

Input and Output: 

Enter a number: 6
Factorial of 6 is 720


*/

File: Python/Day9.py
"""
Day9: Program to find factorial of given number

"""


#Program
n = int(input("Enter a number: "))

factorial = 1
if n < 0:
   print("Sorry, factorial does not exist for negative nbers")
elif n == 0:
   print("The factorial of 0 is 1")
else:
   for i in range(1, n+1):
       factorial = factorial*i
   print("The factorial of",n,"is",factorial)
   
   
"""

Input:
Enter a number: 5

Output:
The factorial of 5 is 120


"""
